influxql/parser.go

```diff
@@ -577,6 +577,15 @@ func (p *Parser) parseSelectStatement(tr targetRequirement) (*SelectStatement, e
 		return nil, err
 	}
 
+	// Set if the query is a raw data query or one with an aggregate
+	stmt.IsRawQuery = true
```

Comment:
In our code we have this concept of "sub statements". Two "select" statements are kicked out from a statement like `SELECT value,mean(value) FROM....` and processing sometimes takes place on sub-statements. Will the `IsRawQuery` flag be set correctly on substatements? I have a feeling that this flag should be a method on the `SelectStatement` type, and *not* determined at parse-time.

cc @benbjohnson